integration-cli/docker_cli_port_test.go

```diff
@@ -18,15 +19,16 @@ func (s *DockerSuite) TestPortList(c *check.C) {
 
 	out, _ = dockerCmd(c, "port", firstID, "80")
 
-	if !assertPortList(c, out, []string{"0.0.0.0:9876"}) {
-		c.Error("Port list is not correct")
-	}
+	err := assertPortList(c, out, []string{"0.0.0.0:9876"})
+	// Port list is not correct
+	c.Assert(err, checker.IsNil, check.Commentf("%v", err))
```

Comment:
No need for this Commentf as `err` will already be in the error output on failure.